Superposition/Tasks.qs

```diff
@@ -118,31 +118,34 @@ namespace Quantum.Kata.Superposition {
 
         // ...
     }
-
-
+    
+    
     // Task 9. Superposition of all basis vectors
     // Input: N qubits in |0...0⟩ state.
     // Goal: create an equal superposition of all basis vectors from |0...0⟩ to |1...1⟩
     // (i.e. state (|0...0⟩ + ... + |1...1⟩) / sqrt(2^N) ).
     operation AllBasisVectorsSuperposition (qs : Qubit[]) : Unit {
         // ...
     }
+    
+    
+	// Task 10. Superposition of all even numbers
+    // Input: N qubits in |0...0⟩ state.
+	// Goal: create a superposition of all even numbers on n qubits
+	operation EvenNumbersSuperposition (qs : Qubit[]) : Unit is Adj {
```

Comment:
Since this is the first time the kata refers to numbers instead of just basis states, you need to provide more details on how the numbers are encoded as basis states. I've found the endian-ness to be a particular challenge, so we'd have to say something like "A basis state represents a number using big-endian binary notation; for example, a state |01⟩ corresponds to an integer 1, and a state |10⟩ - to an integer 2."